server/game/core/gameSteps/prompts/ResourcePrompt.ts

```diff
@@ -86,7 +87,20 @@ export class ResourcePrompt extends AllPlayerPrompt {
     }
 
     public override menuCommand(player, arg): boolean {
-        Contract.fail(`Unexpected menu command: '${arg}'`);
+        if (arg === 'done') {
+            if (this.completionCondition(player)) {
+                return false;
+            }
+            if (this.selectedCards[player.name].length < this.nCardsToResource) {
+                return false;
+            }
+
+            this.resourceSelectedCards(player);
```

Comment:
If I'm reading this right, looks like it will resource the cards for a player as soon as they click done regardless of the other player's finished state. I think we should wait to resource until both players are done.

Reply:
sure i'll change it to when they both complete it!